neo.UnitTests/TestUtils.cs

```diff
@@ -58,13 +58,13 @@ public static void SetupBlockWithValues(Block block, UInt256 val256, out UInt256
             block.Transactions = transactionsVal;
         }
 
-        private static void setupBlockBaseWithValues(BlockBase bb, UInt256 val256, out UInt256 merkRootVal, out UInt160 val160, out uint timestampVal, out uint indexVal, out Witness scriptVal)
+        private static void setupBlockBaseWithValues(BlockBase bb, UInt256 val256, out UInt256 merkRootVal, out UInt160 val160, out long timestampVal, out uint indexVal, out Witness scriptVal)
         {
             bb.PrevHash = val256;
             merkRootVal = UInt256.Parse("0xd841af3d6bd7adb4bca24306725f9aec363edb10de3cafc5f8cca948d7b0290f");
 
             bb.MerkleRoot = merkRootVal;
-            timestampVal = new DateTime(1968, 06, 01, 0, 0, 0, DateTimeKind.Utc).ToTimestamp();
+            timestampVal = new DateTime(1980, 06, 01, 0, 0, 1, 001, DateTimeKind.Utc).ToTimestampMS(); // GMT: Sunday, June 1, 1980 12:00:01.001 AM
```

Comment:
this is a standard date?

Reply:
Yes, 1968 was wrong for timestamp, because unix epoch starts in 1970 (it is in the Helper).

Reply:
Please, answer me instead of hide me ahhahahah

Reply:
Is a random value for the test?